language-server/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -62,17 +97,20 @@ class ScalametaLanguageServer(
   ): List[TextEdit] = {
     try {
       val path = Uri.toPath(td.uri).get
-      val contents = buffers.read(td.uri).getOrElse(read(path))
+      val contents = buffers.read(td.uri).getOrElse(readFromDisk(path))
       val fullDocumentRange = Range(
         start = Position(0, 0),
         end = Position(Int.MaxValue, Int.MaxValue)
       )
-      val formattedContent = scalafmt.format(
-        contents,
-        cwd.resolve(".scalafmt.conf").toString(),
-        path.toString()
-      )
-      List(TextEdit(fullDocumentRange, formattedContent))
+      val config = cwd.resolve(".scalafmt.conf")
+      if (Files.isRegularFile(config.toNIO)) {
+        val formattedContent =
+          scalafmt.format(contents, config.toString(), path.toString())
+        List(TextEdit(fullDocumentRange, formattedContent))
+      } else {
+        connection.showMessage(MessageType.Info, s"Missing $config")
```

Comment:
Can't we just fallback to the default config? I believe that's what most plugins do, right?